docs/self-managed/identity/orchestration-identity/installation.md

```diff
@@ -0,0 +1,53 @@
+---
+id: installation
+title: Installation
+description: "Learn how Identity is bundled with your default Orchestration cluster."
+---
+
+Identity is included by default with the deployment of any [Orchestration cluster](/self-managed/reference-architecture/reference-architecture.md#orchestration-cluster). Within an Orchestration cluster, Identity provides unified, cluster-level identity management and authorizations.
+
+Identity for Orchestration clusters is available via [Helm install](/self-managed/setup/install.md), and for local development via [Camunda 8 Run](/self-managed/setup/deploy/local/c8run.md).
+
+## Initial configuration
+
+Following the default installation will result in a cluster with the following:
+
+1. Basic authentication enabled
+2. API authentication disabled
+3. Authorizations disabled
+4. An initial user with the username/password `demo`/`demo`
+5. An `admin` role with read, create, update, and delete permissions for other roles
+
+### Configure basic authentication
+
+To make changes to the initial configuration, add the desired values via your `application.yaml` or environment variables according to the available [configuration properties](./configuration.md).
+
+For example, to enable basic authentication and create an initial user, the following is required:
```

Comment:
I think this is a little misleading, basic autnetication is the default option so the configuration here is actually about enabling authentication on the API and setting the details of initial users. I think that those topics could be better split so we have one that describes the protected vs unprotected API config and then a second section on initial users that then goes on to explain what the initial users get access to

Reply:
I've divided this section differently in the latest commit, and expanded on the explanations a bit - initial user creation is now explained first, then the protected API and authorizations.